OneCommunityGlobal/HGNRest
# yihan add team code permission

# Description
<img width="721" alt="image" src="https://github.com/OneCommunityGlobal/HGNRest/assets/94303659/2595c9a5-99cc-4f9d-a691-1852b6b64661">

## Related PRS (if any):
The related frontend PR is [#1208](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1208)
This is 2nd PR of #480 which hasn't been merged to dev yet.

## Main changes explained:
- Update file createInitialPermissions.js for adding Edit Team Code permission.

## How to test:
1. check into current branch
2. do `npm run build` and `npm start` to run this PR locally
3. follow test steps of [#1208](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1208)
